src/editing_tools/CropTool.vala

```diff
@@ -41,23 +41,30 @@ public class EditingTools.CropTool : EditingTool {
     private const float MIN_ASPECT_RATIO = 1.0f / 64.0f;
     private const float MAX_ASPECT_RATIO = 64.0f;
 
-    private class ConstraintDescription {
-        public string name;
-        public int basis_width;
-        public int basis_height;
-        public bool is_pivotable;
-        public float aspect_ratio;
+    private class ConstraintDescription : Object {
+        public string name { get; construct; }
```

Comment:
Looking at it, it seems like it should be a class property:
```
public class string name;
```
and set it then in a `static construct`

Reply:
As I am currently a newbie contributor, I would like to ask for help in this one. Can you @tintou help me here? Thank you!